Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -76,6 +76,10 @@ public double translateTypicalShootingPositionToDistance(TypicalShootingPosition
         }
     }
     
+    public double getTargetSpeedForRange(double rangeInInches) {
+        return flushToBoilerTargetSpeed.get() + trimFlushToBoilerSpeed.get();
```

Comment:
hah, good point, it wasn't doing anything before either (presumably a missing conversion of range in inches to different target speeds that we haven't done yet).